tests/unit/type/list.tcl

```diff
@@ -126,30 +83,75 @@ r config set list-max-ziplist-size 1
         assert_equal [r lpop list4] [string repeat c 500]
         assert_equal [r lpop list4] [string repeat b 500]
         assert_equal [r lpop list4] [string repeat a 500]
-    };
+    } {} {needs:debug}
 
     test {plain node check compression with ltrim} {
+        r debug quicklist-packed-threshold 1b
         r lpush list5 [string repeat a 500]
         r linsert list5 after  [string repeat a 500] [string repeat b 500]
         r rpush list5 [string repeat c 500]
         assert_equal [string repeat b 500] [r lindex list5 1]
         r LTRIM list5 1 -1
         r llen list5
-    } {2}
-
+    } {2} {needs:debug}
 
     test {plain node check compression using lset} {
+        r debug quicklist-packed-threshold 1b
         r lpush list6 [string repeat a 500]
         r LSET list6 0 [string repeat b 500]
         assert_equal [string repeat b 500] [r lindex list6 0]
         r lpush list6 [string repeat c 500]
         r LSET list6 0 [string repeat d 500]
         assert_equal [string repeat d 500] [r lindex list6 0]
-    }
+    } {} {needs:debug}
+}
+
+# check functionality of plain nodes using low packed-threshold
+start_server [list overrides [list save ""] ] {
+    # testing lset with combiniations
+    # plain->zip , zip->plain, plain->plain, zip->zip
+    test {plain check LSET with combinations} {
```

Comment:
i didn't understand what "combinations" means, now the comment above explains it, but maybe a better name could be:
`Test LSET with packed / plain, old / new combinations`?
i'm also willing to leave it as is, but either way, better keep the comment above and if needed update other tests / comments that refer to this one..